WooCommerce/Classes/Authentication/Navigation Exceptions/ULAccountMismatchViewController.swift

```diff
@@ -154,7 +156,9 @@ private extension ULAccountMismatchViewController {
 
     func configureExtraInfoButton() {
         extraInfoButton.applyLinkButtonStyle()
-        extraInfoButton.contentEdgeInsets = Constants.extraInfoCustomInsets
+        var configuration = UIButton.Configuration.plain()
+        configuration.contentInsets = Constants.extraInfoCustomInsets
+        extraInfoButton.configuration = configuration
```

Comment:
Similar as above, you should set the custom inset to the existing configuration:

```suggestion
        extraInfoButton.configuration.contentInsets = Constants.extraInfoCustomInsets
```